header.css

```diff
@@ -0,0 +1,282 @@
+.header {
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+    position: relative;
+    padding-top: 23px;
+    margin-bottom: 34px;
+    box-sizing: border-box;
+    height: 96px;
+}
+@media screen and (min-width: 1026px) and (max-width: 1171px) {
+    .header {
+        margin-bottom: 18px;
+    }
+}
+@media screen and (max-width: 1023px) {
+    .header {
+        margin-bottom: 0;
+    }
+}
+@media screen and (min-width: 767px) and (max-width: 1023px) {
+    .header {
+        height: 112px;
+    }
+}
+@media screen and (min-width: 375px) and (max-width: 766px) {
+    .header {
+        height: 73px;
+    }
+}
+@media screen and (max-width: 374px) {
+    .header {
+        height: 71px;
+    }
+}
+@media screen and (min-width: 767px) and (max-width: 1023px) {
+    .header {
+        padding-top: 8px; 
+    }
+}
+@media screen and (max-width: 766px) {
+    .header {
+        padding-top: 0; 
+    }
+}
+.header__logo {
+    width: 61px;
+    height: 55px;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/gh/WebHeroSchool/final-Axion/pullRequest?prid=5455661)